src/main/scala/tech/cryptonomic/conseil/Conseil.scala

```diff
@@ -8,13 +8,15 @@ import akka.http.scaladsl.server.Directives._
 import akka.stream.ActorMaterializer
 import ch.megard.akka.http.cors.scaladsl.CorsDirectives._
 import com.typesafe.scalalogging.LazyLogging
+import de.heikoseeberger.akkahttpcirce.FailFastCirceSupport
 import tech.cryptonomic.conseil.config.ConseilAppConfig
 import tech.cryptonomic.conseil.directives.EnableCORSDirectives
 import tech.cryptonomic.conseil.routes._
+import tech.cryptonomic.conseil.routes.openapi.OpenApiDoc
 
 import scala.concurrent.ExecutionContextExecutor
 
-object Conseil extends App with LazyLogging with EnableCORSDirectives with ConseilAppConfig {
+object Conseil extends App with LazyLogging with EnableCORSDirectives with ConseilAppConfig with FailFastCirceSupport {
```

Comment:
request for info: how does `FailFast` work wrt to any other choice?

Reply:
there is two of them: `FailFast` and `ErrorAccumulating`, the difference is that the second one accumulates all of the errors during deserialization: https://github.com/hseeberger/akka-http-json/blob/master/akka-http-circe/src/main/scala/de/heikoseeberger/akkahttpcirce/CirceSupport.scala